opentelemetry-sdk/src/opentelemetry/sdk/trace/export/__init__.py

```diff
@@ -228,7 +230,10 @@ def on_end(self, span: ReadableSpan) -> None:
             if not self._spans_dropped:
                 logger.warning("Queue is full, likely spans will be dropped.")
                 self._spans_dropped = True
-
+                self._dropped_spans_counter = Meter.create_counter(
```

Comment:
I think there's a bug here (not related to your change). self._spans_dropped should be reset to `False` after the queue has flushed some items. In this case, I would not rely on `self._spans_dropped` as the indicator of whether to create the counter, but instead if the counter exists already.